docs/application/dotnet/guides/user-interface/nui/absolute-layout.md

```diff
@@ -0,0 +1,182 @@
+<style>
+    .tabcontent img {
+        border: 1px solid #555;
+        max-width: 100% !important;
+        max-height: 100%;
+    }
+</style>
+
+# Absolute Layout
+
+`AbsoluteLayout`, allowing explicit positioning of children. Positions are from the top left of the layout and can be set using the `View.Position` and alike. `AbsoluteLayout` is default layout for every new view.
```

Comment:
`AbsoluteLayout` allows to explicit positioning of the children components. The positions are defined from the top left cornet of the layout and can be set using the `View.Positions` property.

Reply:
Done